### PR TITLE
chore: made controller tests more explicit

### PR DESCRIPTION
**Pull request by igpetrov**
chore: made controller tests more explicit

### CODE REVIEW

File: connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerTestZeebeTests.java [hunk 1]
```diff
@@ -205,6 +212,44 @@ public void testSuccessfulProcessingWithFailedActivation() throws Exception {
...
+                            "myPath",
+                            new HashMap<>(),
+                            "{}".getBytes(),
+                            new HashMap<>(),
+                            new MockHttpServletRequest());
+
+    assertEquals(200, responseEntity.getStatusCode().value());
+
+    var result = responseEntity.getBody();
+    assertNull(result);
```
Comment (sbuettner) [inline connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerTestZeebeTests.java:252]: Would it make sense to json assert the response?

Comment (igpetrov) [reply connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerTestZeebeTests.java:252]: it's null :)

as per contract - if no response body expression given, we just return `200 OK` and no body. Json assert won't be different here.

**Review by chillleader — APPROVED** [review]

File: connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerTestZeebeTests.java (hunk 1 above)
Comment (sbuettner) [reply connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerTestZeebeTests.java:254]: Right, I meant to add a case where its not null.

Comment (igpetrov) [reply connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerTestZeebeTests.java:254]: I see your point. For that case we do parsing already, like:

```
ResponseEntity<InboundConnectorResult<?>> responseEntity =
        (ResponseEntity<InboundConnectorResult<?>>)
            controller.inbound(
                "myPath",
                new HashMap<>(),
                "{}".getBytes(),
                new HashMap<>(),
                new MockHttpServletRequest());
```

And I added those use-cases as asserts by actual object properties.

Comment (sbuettner) [reply connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerTestZeebeTests.java:254]: I meant a real test against the actual rendered json response and not the `ResponseEntity`.

Comment (igpetrov) [reply connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerTestZeebeTests.java:254]: But we don't have these now. Or at least I am not aware of. If so, I would suggest to have a separate task that I can take after I am back. Today is the last day before I leave for FTO, and there are more urgent tasks left - regression testing and documentation.

Comment (sbuettner) [reply connector-runtime/spring-boot-starter-camunda-connectors/src/test/java/io/camunda/connector/runtime/inbound/WebhookControllerTestZeebeTests.java:254]: Alright.

**Review by sbuettner — APPROVED** [review]

**Pull request merged**